### PR TITLE
FWI-2260 - Validate OPA policies in our library during insights-plugins CI

### PR DESCRIPTION
**Pull request by ivanfetch-fw**
## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### What's the goal of this PR?
Use the Insights CLI to validate OPA policies modified compared to the `main` branch.

### Example Output

Only policies which have changed are validated. In this example, most policies have been modified to add test Kube manifests to them. Here is an example of a failed validation job [in CI](https://app.circleci.com/pipelines/github/FairwindsOps/insights-plugins/2816/workflows/d9398dad-c3fa-43e9-9810-e6cfab35cb5a/jobs/8074) and [in Slack](https://fairwindsops.slack.com/archives/CPYBX810T/p1657104760978769).

[Internal Ticket FWI-2260](https://fairwinds.myjetbrains.com/youtrack/issue/FWI-2260)

### CODE REVIEW

File: plugins/opa/examples-v2/label-blocked/policy.success.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# This manifest is used to test the accompanying OPA policy.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: policy-test
+  namespace: default
+  labels:
+    app: myapp
+     # Lack of this label will cause the policy to fail.
```
[the remark this answers is not in the document]
Comment (makoscafee) [reply plugins/opa/examples-v2/label-blocked/policy.success.yaml:9]: or if the comment is true... the file name says success.yaml

[the remark this answers is not in the document]
Comment (ivanfetch-fw) [reply plugins/opa/examples-v2/label-blocked/policy.success.yaml:9]: I updated the comment to clarify that lack of the *below* label causes the policy to `s/fail/succeed` - thank you.

**Comment by ivanfetch-fw** [comment]
@makoscafee Thanks for the thorough review and catching those mistakes!

**Comment by ivanfetch-fw** [comment]
I also updated the `examples-v2/README.md` to mention policy testing, and request that new policies have tests added.

**Pull request merged**